dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java

```diff
@@ -648,8 +822,7 @@ public boolean mkdirs() throws Exception {
      *     ever be {@link InterruptedException}.
      */
     public static void sleep(long ns) throws Exception {
-        Thread.currentThread().sleep( (long) (ns / 1000), (int) (ns % 1000) );
-        // throws InterruptedException;
+        Thread.sleep( (long) (ns / 1000), (int) (ns % 1000) ); // throws InterruptedException;
```

Comment:
this could be simplified to `TimeUnit.NANOSECONDS.sleep(ns)`

Reply:
Replaced with simplified API.